app/models/package.rb

```diff
@@ -258,7 +259,7 @@ def designation
   end
 
   def cancel_designation
-    designation and designation.cancel!
+    designation ** designation.cancel!
```

Comment:
Is ** correct here?

Reply:
Its wrong. It should be &&. Thanks @steveyken 👍